src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -95,10 +95,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + feedbackResponseComment.feedbackSessionName + "<br>"
                            + "by: " + feedbackResponseComment.giverEmail + "<br>"
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
+            // createdAt, lastEditedAt, commentGiverName and commentEditorName are required to generate editedCommentDetails
```

Comment:
Apologies; one last thing. On second thoughts, the part about `commentGiverName` and `commentEditorName` is unnecessary, as it is already clear from the fact that these are passed into the `createEditedCommentDetails` method.

Move this comment to just before the line `data.comment = updatedComment;` and change it to `// createdAt and lastEditedAt fields in updatedComment as well as sessionTimeZone are required to generate timestamps in editedCommentDetails`.